waspc/data/Generator/templates/server/src/entities/index.ts

```diff
@@ -0,0 +1,17 @@
+{{={= =}=}}
+import {
+  {=# entities =}
+  {= name =},
+  {=/ entities =}
+} from '@prisma/client'
+  
+export {
+  {=# entities =}
+  {= name =},
+  {=/ entities =}
+} from '@prisma/client'
+
+export type WaspEntity = 
```

Comment:
Aha, this is just some kind of sum type that unifies all the entity types together?

Reply:
Yes, that's exactly what it is.